### PR TITLE
Added alignments

### PR DESCRIPTION
**Pull request by shaharkazaz**
Also, I suggest removing d-flex, since the default dir is row, we should use d-flex-row to make it more clear

### CODE REVIEW

**Comment by NetanelBasal** [comment]
Are you sure it's not breaking anything?

**Comment by shaharkazaz** [comment]
As soon as we decide to remove it I will update all the uses in our system.
If you want we can add a deprecated decoration to appear in the sass doc
and remove it step by step

**Comment by NetanelBasal** [comment]
I think  flex-direction: row; is redundant, this is the default.

**Comment by NetanelBasal** [comment]
`<div class="d-flex"></div>` should be `display: flex` and that's it.

File: atomic-css/src/flex.scss [hunk 1]
```diff
@@ -209,20 +204,33 @@
   align-items: center;
 }
 
+.align-center-end {
+  place-content: center;
```
Comment (NetanelBasal) [inline atomic-css/src/flex.scss]: there is no need for place-content in all the three places, remove it, please.

Comment (shaharkazaz) [reply atomic-css/src/flex.scss]: what do you mean? I can replace with justify-content

Comment (NetanelBasal) [reply atomic-css/src/flex.scss]: place content is a shortcut for align and justify, but here you just need align-items

**Comment by shaharkazaz** [comment]
ok so just to be clear, we are removing the d-flex-row and keeping the d-flex.
I agree that behind the scene it should only be `display: flex` but I think we should keep the 'd-flex-row' naming instead of just d-flex, it's 1 word but you can't be any clearer

**Comment by NetanelBasal** [comment]
I still think it's redundant, when you write display flex you already know that's its row. We are using this class a lot in the application so it's better to keep it short.

File: atomic-css/src/flex.scss (hunk 1 above)
Comment (shaharkazaz) [reply atomic-css/src/flex.scss]: but this is center-end,if you don't add the justify contetnt the items would stay in thier normal alignment

**Pull request merged**